packages/terra-form-select/src/native-select/NativeSelect.module.scss

```diff
@@ -0,0 +1,173 @@
+      color: var(--terra-form-select-native-hover-color, #1c1f21);
+    }
+
+    &.incomplete {
+      background-color: var(--terra-form-select-native-incomplete-background-color, #ffc);
+    }
+
+    &.invalid {
+      background-color: var(--terra-form-select-native-invalid-background-color, none);
+      background-image: var(--terra-form-select-native-invalid-background-image, none);
+      border-color: var(--terra-form-select-native-invalid-border-color, #e50000);
+      box-shadow: var(--terra-form-select-native-invalid-box-shadow, 0 0 0 1px #e50000);
+
+      .arrow-icon {
+        background-image: var(--terra-form-select-native-invalid-arrow-icon-background-image, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" fill="#e50000" class="is-bidi"><path d="M48 12L24 36 0 12h48z"/></svg>'));
+      }
+
+      &[data-focus-interaction='keyboard'] {
+        box-shadow: var(--terra-form-select-native-invalid-focus-keyboard-box-shadow, 0 0 0 1px #e50000);
+      }
+  
+      &[data-focus-interaction='keyboard'],
+      &[data-focus-interaction='mouse'], {
+        border-color: var(--terra-form-select-native-invalid-focus-border-color, #e50000);
+      }
+  
+      &:hover {
+        border-color: var(--terra-form-select-native-invalid-hover-border-color, #e50000);
+      }
+    }
+
+    &.placeholder {
+      color: var(--terra-form-select-native-placeholder-color, rgba(21, 33, 41, 0.62));
+      font-size: var(--terra-form-select-native-placeholder-font-size, 1.143rem);
+      font-style: var(--terra-form-select-native-placeholder-font-style, normal);
+    }
+
+    &.disabled {
+      background-color: var(--terra-form-select-native-disabled-background-color, rgba(111, 116, 119, 0.2));
+      background-image: none;
+      border-color: var(--terra-form-select-native-disabled-border-color, #dedfe0);
+      color: var(--terra-form-select-native-disabled-color, #1c1f21);
+      opacity: var(--terra-form-select-native-disabled-opacity, 0.4);
+
+      .arrow-icon {
+        background-image: var(--terra-form-select-native-disabled-arrow-icon-background-image, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" fill="rgba(17, 22, 25, .5)" class="is-bidi"><path d="M48 12L24 36 0 12h48z"/></svg>'));
+      }
+    }
+  }
+
+  .display {
+    align-items: center;
+    background-color: transparent;
+    flex: 1 1 auto;
+    min-width: 0;
+    overflow: hidden;
+    position: relative;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    width: 100%;
+    word-break: normal;
+    word-wrap: normal;
+  }
+
+  .frame {
+    align-items: center;
+    display: flex;
+    flex: 1 1 auto;
+    height: 100%;
+    left: 0;
+    max-width: 100%;
+    min-width: 0;
+    padding-bottom: 0;
+    padding-left: var(--terra-form-select-native-frame-padding-left, 0.4286rem);
+    padding-right: 0;
+    padding-top: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: 0;
+  }
+
+  .select {
```

Comment:
The positioning seems a little misaligned on the deployed site. Is this the intended position of the dropdown when open?

![image](https://user-images.githubusercontent.com/6720991/83557108-aec00d80-a4d6-11ea-92c5-b63bb296716a.png)


Slightly offset to the left, a small reveal of the dropdown arrow

Reply:
This occurs depending on the browser. When altering the dropdown to span 100% of the width it breaks certain browsers alignment. After some discussion with UX, while it is annoying it acceptable.